diff --git a/node_modules/playwright-core/lib/server/firefox/ffNetworkManager.js b/node_modules/playwright-core/lib/server/firefox/ffNetworkManager.js
index bbadf37..cbaa411 100644
--- a/node_modules/playwright-core/lib/server/firefox/ffNetworkManager.js
+++ b/node_modules/playwright-core/lib/server/firefox/ffNetworkManager.js
@@ -42,9 +42,7 @@ class FFNetworkManager {
   async setRequestInterception(enabled) {
     await Promise.all([this._session.send('Network.setRequestInterception', {
       enabled
-    }), this._session.send('Page.setCacheDisabled', {
-      cacheDisabled: enabled
-    })]);
+    }), ]);
   }
   _onRequestWillBeSent(event) {
     const redirectedFrom = event.redirectedFrom ? this._requests.get(event.redirectedFrom) || null : null;
